app/src/main/java/com/xmartlabs/template/helper/DatabaseHelper.java

```diff
@@ -22,10 +23,9 @@ public void deleteAll() {
   }
 
   public void migrate(@NonNull Session session) {
```

Comment:
If you guys think this helper is useful as is, leave it. But consider making it abstract or something because now it's on a library